src/redux/Slice.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] It will be great to fix the eslint errors instead of disabling the rule. Remember, eslint is there to ensure that you write error-free code. Kindy fix this issue